src/app.js

```diff
@@ -26,14 +26,18 @@ const chaosExperimentsManager = require('./chaos-experiments/models/chaosExperim
 const streamingManager = require('./streaming/manager');
 const streamingConfig = require('./config/streamingConfig');
 const contexts = require('./middlewares/context');
+const configHandler = require('./configManager/models/configHandler');
 
 module.exports = async () => {
     swaggerValidator.init('./docs/openapi3.yaml', { beautifyErrors: true });
     await database.init();
     await jobsManager.init();
     await jobsManager.reloadCronJobs();
     await jobsManager.scheduleFinishedContainersCleanup();
-    await chaosExperimentsManager.init();
+    const { chaos_mesh_enabled: isChaosEnabled } = await configHandler.getConfig();
+    if (isChaosEnabled){
+        await chaosExperimentsManager.init();
```

Comment:
let's put it in one place to avoid to if condtions
```
if (isChaosEnabled){
        await chaosExperimentsManager.init();
       app.use('/v1/chaos-experiments', chaosExperimentsRouter);
```

Reply:
done